lib/tmuxinator/config.rb

```diff
@@ -42,10 +44,21 @@ def exists?(name)
         File.exists?(project(name))
       end
 
-      def project(name)
+      def project_in_root(name)
         projects = Dir.glob("#{root}/**/*.yml")
-        project_file = projects.detect { |project| File.basename(project, ".yml") == name }
-        project_file || "#{root}/#{name}.yml"
+        projects.detect { |project| File.basename(project, ".yml") == name }
+      end
+
+      def project_in_local(name)
```

Comment:
Unused method argument - `name`. If it's necessary, use `_` or `_name` as an argument name to indicate that it won't be used. You can also write as `project_in_local(*)` if you want the method to accept any arguments but don't care about them.